### PR TITLE
Id877 test recv func

### PR DESCRIPTION
**Pull request by AmirLevant**
I added the trim line which fixed the issue with the URI portion splitting improperly, receive function should work good now.

### CODE REVIEW

File: scriptOutput.log
Comment (sebastian-j-ibanez) [inline scriptOutput.log:1]: This should be untracked. We can add it to the gitignore.

**Review by Inventhrice — APPROVED** [review]
Looks good!

File: telemetry.json
Comment (sebastian-j-ibanez) [inline telemetry.json:1]: Honestly we should add this file to the gitignore as well.

File: server
Comment (sebastian-j-ibanez) [inline server:1]: Where did this come from? No idea how you generated it but it should be untracked from the repo (or just deleted).

File: test/test.http [hunk 1]
```diff
@@ -20,3 +20,39 @@ GET http://localhost:8080/styles/styles.css
 ###
 
 GET http://localhost:8080/execute/testScriptName
+
+####
+// bad test do not test
```
Comment (sebastian-j-ibanez) [inline test/test.http:25]: You've add the quotations to this test, so it will work. You can remove the comment about it being bad.

**Review by sebastian-j-ibanez — CHANGES_REQUESTED** [review]
I have added my comments to the PR. Looks like there's a couple things that need to be added to the gitignore. 

Aside from that, it looks good.

**Pull request merged**